ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -174,53 +255,72 @@ server {
         }
     }
 
+    # Redirects the Legacy /documentation endpoint to /api
+    # We use ^~ to tell NGINX to not process any other Location directive or Rewrite after this match
     location ^~ /documentation {
         rewrite ^/documentation/api(.*)$ /api$1 redirect;
     }
 
+    # This Location is used for our Metrics Pages
+    # We keep `autoindex` on as some folders are autoindex (do not have an index.html)
+    # even tho the main /metric endpoint is a build page (statically generated over time)
+    # We use ^~ to tell NGINX to not process any other Location directive or Rewrite after this match
     location ^~ /metrics {
         alias /home/dist/metrics;
         autoindex on;
         default_type text/plain;
     }
 
-    location = /github-webhook.log {
-        alias /home/nodejs/github-webhook.log;
-        default_type text/plain;
-    }
-
+    # Store the original language of the request
+    # We'll use this for the 404 in the try_files in @localized_404
     location @english_fallback {
-        # Store the original language of the request
-        # We'll use this for the 404 in the try_files in @localized_404
+        # @TODO: Handle Localization Fallback through Next.js SSR as this is a hacky approach and requires
+        # continuous maintenance of the supported languages
         if ($uri ~* ^/(ar|be|ca|de|es|fa|fr|gl|id|it|ja|ka|ko|nl|pt-br|ro|ru|tr|uk|zh-cn|zh-tw)/) {
             set $lang $1;
         }
 
-        rewrite ^/(ar|be|ca|de|es|fa|fr|gl|id|it|ja|ka|ko|nl|pt-br|ro|ru|tr|uk|zh-cn|zh-tw)/(.*)$ /en/$2 last;
+        rewrite ^/(ar|be|ca|de|es|fa|fr|gl|id|it|ja|ka|ko|nl|pt-br|ro|ru|tr|uk|zh-cn|zh-tw)/(.*)$ /en/$2;
     }
 
+    # This Location directive is used for handling all 404's requests coming to this Server
+    # by default if the request doesn't come from a /two-letter-code/ localized page (extracted from the @english_fallback Location)
+    # it will then fallback to $lang as being "en" as defined numerous lines above
```

Comment:
```suggestion
    # This location directive handles all 404 responses for the server
    # If the request was a localised website page, use the requested language
    # as set by the @english_fallback location block
    # Otherwise, this will fallback to $lang being "en" as defined numerous lines above
```